### PR TITLE
Past proof polyfill/transpiler - WIP

### PR DESCRIPTION
**Pull request by fuentesloic**
### Context

Old people with old browser, old mind and old laptop/computer should be able to use Lego.js

### Comment
Looking for feedback as I'm not sure the good way to get done this.

### Need to do
- Define browsers support

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,18 +7,28 @@
...
   "scripts": {
     "test": "mocha --require @babel/register --require @babel/polyfill",
     "test-debug": "npm test -- debug -R spec",
-    "compile": "LEGO_PATH=/lib/index.js ./bin/compile.js ./demo/components/ ./demo/components.js",
-    "watch": "npm run compile -- -w"
+    "compile": "LEGO_PATH=../lib/index.js ./bin/compile.js ./demo/components/ ./demo/components.js",
+    "browserify": "browserify ./demo/components.js -o ./demo/components.js -t [ babelify --presets [ @babel/preset-env ] --plugins [ @babel/plugin-proposal-class-properties @babel/plugin-proposal-export-default-from @babel/plugin-proposal-export-namespace-from] ]",
+    "compile-past": "npm run compile && npm run browserify",
+    "watch": "npm run compile -- -w",
+    "serve": "python3 -m http.server 8080"
```
Comment (vinyll) [inline package.json]: I'm not sure if we should depend on python3 here.
This is only required to run the demo.
Eventually we could suggest in the doc to:

```
npm install http-server
http-server
open http://localhost:3000
```
WDYT?

File: .gitignore [hunk 2]
```diff
@@ -1,2 +1 @@
 /node_modules/
```
Comment (vinyll) [inline .gitignore]: Man, you still haven't configure your code editor to keep or add the last line empty on files…

File: package.json (hunk 1 above)
Comment (fuentesloic) [reply package.json]: you right!

File: .gitignore (hunk 2 above)
Comment (fuentesloic) [reply .gitignore]: not my laptop ^^

Comment (newick) [reply .gitignore]: Your suggestion isn't very inclusive ;)
Another solution: https://editorconfig.org/

Comment (fuentesloic) [reply .gitignore]: `.editorconfig` init

**Pull request closed without merging**